### DISCUSSION TITLE
Update Trigger Timers

### DISCUSSION THREAD

**Discussion by elusive-ttv** [gameserverapp/Platform#100] [Suggestions]
![image](https://user-images.githubusercontent.com/100104764/155869505-be7369aa-294c-4a9b-9347-8f8abf629947.png)

I would love to see 5 min increments available, at least up to the 45 minute mark. If it would be possible, it would make our cluster update so much smoother. <3 Thank you for your consideration!

**Comment by mvaessen** [comment]
created development item #104
